### PR TITLE
Fix docs link

### PR DESCRIPTION
**Pull request by millerdev**
Self-merging.

### CODE REVIEW

File: docs/migrations.rst [hunk 1]
```diff
@@ -86,4 +86,4 @@ to ensure we can rebuild the view without issue.
 Migration Patterns and Best Practices
 -------------------------------------
 
-- `Auto-Managed Migration Pattern <migration_command_pattern>`_.
+- :ref:`auto-managed-migration-pattern`
```
Comment (emord) [inline docs/migrations.rst]: @proteusvacuum i think we were trying to figure this out a couple week ago. fyi in case you're still curious

Comment (proteusvacuum) [reply docs/migrations.rst]: I tried many variations of this but couldn't get it to work... happy someone figured out the right syntax.

Comment (millerdev) [reply docs/migrations.rst]: The [docs build failed](https://readthedocs.org/projects/commcare-hq/builds/8744531/), so I don't know if it works yet.

Comment (proteusvacuum) [reply docs/migrations.rst]: You can test locally with:
```
$ pip install -r requirements/docs-requirements.txt
$ cd docs
$ make html
```

Comment (millerdev) [reply docs/migrations.rst]: build passed and it works

**Pull request merged**